jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/upload/EditFunctionDialog.kt

```diff
@@ -146,7 +146,9 @@ class EditFunctionDialog(
         if (mode == UPDATE_CODE) {
             UIUtil.uiChildren(view.configurationSettings).filter { it !== view.handler && it !== view.handlerLabel }.forEach { it.isVisible = false }
         }
-        view.runtime.populateValues(default = runtime) { Runtime.knownValues() }
+
+        view.setRuntimes(Runtime.knownValues())
+        view.runtime.selectedItem = runtime
```

Comment:
This should be `validOrNull'ed` to handle if the runtime is newer than the SDK so that it doesnt change to a different value but is instead blank

Reply:
Oops, done.